frontend/javascripts/oxalis/model_initialization.ts

```diff
@@ -700,11 +700,11 @@ async function applyLayerState(stateByLayer: UrlStateByLayer) {
       // The name of the layer could have changed if a volume tracing was created from a viewed annotation
       effectiveLayerName = getLayerByName(dataset, layerName, true).name;
     } catch (e) {
-      console.error(e);
```

Comment:
In cases like this I didnt omit the separate log because Error not only has a message, but also a type, and I didnt want that to be omitted. Let me know if you think I should change that :)